plugins/browser-plugin-form-tracking/src/helpers.ts

```diff
@@ -90,12 +60,18 @@ export interface ElementData extends Record<string, string | null | undefined> {
   type?: string;
 }
 
-export type transformFn = (x: string | null, elt: ElementData | TrackedHTMLElement) => string | null;
+export type transformFn = (
+  elementValue: string | null,
+  elementInfo: ElementData | TrackedHTMLElement,
+  elt: TrackedHTMLElement
```

Comment:
In my view it is clearer to allow for an 'expected' input _(even if in v3 it is a clunky third attribute)_ rather than trying to explain our, mistaken in my opinion, logic of how the `elt` attribute would work. 
I would much prefer adding the option of 'here is your element that you know how to work with and its properties are as expected' rather than changing an already weird logic _(the getElementIdentifier)_.

Reply:
Okay, happy to go with that, but please make it clear in the docs what to expect when.